build.sbt

```diff
@@ -135,4 +135,7 @@ buildWebapp := {
   Process(s"npm run build -- -p --env.out=${crossTarget.value}/classes") !
 }
 
+// Disable parallel execution until it stops causing deadlocks with Mockito
+parallelExecution := false
```

Comment:
I think it's better to put it in root settings and limit its scope to `parallelExecution in Test`

Reply:
Makes sense, updated it.